stdlib/http/src/main/java/org/ballerinalang/net/http/nativeimpl/ParseHeader.java

```diff
@@ -87,7 +87,7 @@ public static Object parseHeader(Strand strand, String headerValue) {
 
         // set parse error
         ErrorValue mimeError = MimeUtil.createError(READING_HEADER_FAILED, errMsg);
-        String httpErrorMessage = "MimeError occurred while parsing the header";
+        String httpErrorMessage = "mimeError occurred while parsing the header";
```

Comment:
sounds odd to say "mimeError occurred". Doesn't it suffice to say "failed to parse the header" or something?